File: Chat.UI/src/models/message.js
export default class Message {
    chatId;
    messageContent;
    senderId;
    senderName;
    sendDate;

    constructor(chatId, messageContent, senderId, senderName, sendDate) {
        this.chatId = chatId;
        this.senderName = senderName;
        this.senderId = senderId;
        this.senderName = senderName;
        this.messageContent = messageContent;
        this.sendDate = sendDate;
    }
}

File: Chat.API/Chat.API/Domain/Entities/Message.cs
using System.ComponentModel.DataAnnotations.Schema;
using Chat.API.Domain.BaseEntities;

namespace Chat.API.Domain.Entities;

public class Message : BaseAuditableEntity
{
    [ForeignKey(nameof(User))]
    public Guid UserId { get; set; }
    
    public User User { get; set; } = null!;
    
    [ForeignKey(nameof(Chat))]
    public Guid ChatId { get; set; }
    
    public Chat Chat { get; set; } = null!;
    
    public string MessageContent { get; set; } = string.Empty;
}